app/lpc55xpresso/stage0.toml

```diff
@@ -8,9 +8,9 @@ external-images = ["a", "b"]
 
 [kernel]
 name = "stage0"
-requires = {flash = 0x7000, ram = 8192}
+requires = {flash = 0x7000, ram = 16000}
```

Comment:
I'm guessing this bump is because we have yet another certificate?

Reply:
Yes. Currently I'm cloning the whole cert chain to serialize certs into each handoff block since the serialization framework requires the serialized struct to own all serialized data. This probably won't be sustainable so I'm looking for ways to reduce the data duplication. The only thing I've come up with so far is to create a separate region in the handoff block to hold the common certs (the intermediate CA and the DeviceId). I'm :100: open to alternative solutions if you've got ideas though.